db/migrate/20210817100257_rename_stripe_connect_payment_methods_to_stripe_sca.rb

```diff
@@ -0,0 +1,7 @@
+class RenameStripeConnectPaymentMethodsToStripeSca < ActiveRecord::Migration[6.1]
+  def change
+    Spree::PaymentMethod
+      .where(type: "Spree::Gateway::StripeConnect")
+      .update_all(type: "Spree::Gateway::StripeSCA")
```

Comment:
Migrations last longer if they avoid using app code. In this case, it would be better to avoid `Spree::PaymentMethod` and execute simple SQL. While this statement looks simple, it doesn't actually replace all records:

```rb
puts Spree::PaymentMethod.where(type: "Spree::Gateway::StripeConnect").to_sql
SELECT "spree_payment_methods".* FROM "spree_payment_methods" WHERE \
"spree_payment_methods"."deleted_at" IS NULL AND "spree_payment_methods"."deleted_at" IS NULL AND \
"spree_payment_methods"."type" = 'Spree::Gateway::StripeConnect'
```

As you can see, there is a default scope that ignores deleted payment methods. If we changed the app code later and tried to load deleted payment methods then the app would crash on those old rows.

As a general rule: App code changes constantly but migrations should stay the same. So they should be independent and not be mixed.

```suggestion
    execute "UPDATE spree_payment_methods SET type = 'Spree::Gateway::StripeSCA' WHERE type = 'Spree::Gateway::StripeConnect'"
```